core-aam/aria-valuenow-spinbutton-default-manual.html

```diff
@@ -0,0 +1,76 @@
+<!doctype html>
+<html>
+  <head>
+    <title>aria-valuenow</title>
+    <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
+    <link rel="stylesheet" href="/wai-aria/scripts/manual.css">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/wai-aria/scripts/ATTAcomm.js"></script>
+    <script>
+    setup({explicit_timeout: true, explicit_done: true });
+
+    var theTest = new ATTAcomm(
+    {
+   "steps" : [
+      {
+         "element" : "test",
+         "test" : {
+            "ATK" : [
+               [
+                  "result",
+                  "atk_value_get_current_value()",
+                  "is",
+                  "0"
+               ]
+            ],
+            "AXAPI" : [
+               [
+                  "property",
+                  "AXValue",
+                  "is",
+                  ""
+               ]
+            ],
+            "IAccessible2" : [
+               [
+                  "property",
+                  "currentValue",
+                  "is",
+                  "VT_EMPTY"
+               ],
+               [
+                  "property",
+                  "accValue",
+                  "is",
+                  ""
+               ]
+            ],
+            "UIA" : [
+               [
+                  "property",
+                  "RangeValue.Value",
+                  "is",
+                  "0"
```

Comment:
I think this property can't be anything other than an double: https://docs.microsoft.com/en-us/windows/win32/api/uiautomationcore/nf-uiautomationcore-irangevalueprovider-get_value

Reply:
Chrome implementation: https://source.chromium.org/chromium/chromium/src/+/main:ui/accessibility/platform/ax_platform_node_win.cc;l=3358?q=accValue&ss=chromium%2Fchromium%2Fsrc